src/request/sign-transaction/SignTransaction.css

```
@@ -1,154 +1,180 @@
+    align-items: center;
 }
 
-.transaction .total .nim-symbol {
-    margin-left: 1rem;
+#confirm-transaction #effect-container {
+    display: flex;
+    flex-direction: column;
+    overflow: hidden;
+    border-radius: 1rem;
+    flex-grow: 1;
 }
 
-.transaction .fee-section {
-    text-align: center;
-    opacity: 0.5;
+#confirm-transaction > #effect-container > * {
+    transition: filter .4s, opacity .4s;
+    -webkit-filter: blur(0px);
+    -moz-filter: blur(0px);
+    -o-filter: blur(0px);
+    -ms-filter: blur(0px);
+    filter: blur(0px);
+    opacity: 1;
 }
 
-.transaction .data-section {
-    margin: 2rem 3rem 0;
-    text-align: center;
+#confirm-transaction.account-details-open > #effect-container > .page-header,
+#confirm-transaction.account-details-open > #effect-container > .page-body {
+    -webkit-filter: blur(20px);
+    -moz-filter: blur(20px);
+    -o-filter: blur(20px);
+    -ms-filter: blur(20px);
+    filter: blur(20px);
+    opacity: 0.5;
 }
 
-.layout-standard .nq-card-footer {
-    border-radius: 0;
+#confirm-transaction.account-details-open > #effect-container > .page-footer {
+    -webkit-filter: blur(35px);
+    -moz-filter: blur(35px);
+    -o-filter: blur(35px);
+    -ms-filter: blur(35px);
+    filter: blur(35px);
+    opacity: 0.5;
 }
 
-.page-footer {
-    padding: 1rem;
+#confirm-transaction > #background-overlay {
+    z-index: -3;
+    position: fixed;
+    top: 0;
+    left: 0;
+    bottom: 0;
+    right: 0;
 }
 
-.layout-standard ~ .page-footer {
-    background: white;
-    border-bottom-left-radius: 1rem;
-    border-bottom-right-radius: 1rem;
+#confirm-transaction.account-details-open > #background-overlay {
+    z-index: 2;
 }
 
-/*
-#######################################
-#### C H E C K O U T   L A Y O U T ####
-#######################################
-*/
-
-.payment-info-line {
-    margin-top: -3.5rem;
+#confirm-transaction #account-details #close-details {
+    position: absolute;
+    right: 2rem;
+    top: 2rem;
+    width: 3rem;
+    height: 3rem;
+    opacity: 0.16;
+    border: 0rem;
+    background-color: transparent;
+    background-image: url("data:image/svg+xml;charset=utf-8,<svg width='38' height='38' fill='none' xmlns='http://www.w3.org/2000/svg'><path fill-rule='evenodd' clip-rule='evenodd' d='M9.808 9.808c5.068-5.069 13.316-5.069 18.384 0 5.069 5.068 5.069 13.316 0 18.384-5.068 5.069-13.316 5.069-18.384 0-5.069-5.068-5.069-13.316 0-18.384zM19 17.586l3.535-3.536 1.415 1.414L20.414 19l3.536 3.535-1.415 1.415L19 20.414l-3.536 3.536-1.414-1.415L17.586 19l-3.536-3.536 1.414-1.414L19 17.586z' fill='%23101531'/></svg>");
+    background-size: 135%;
 }
 
-.transaction-section {
-    background: var(--nimiq-card-header-bg);
-    border-bottom: 1px solid var(--nimiq-card-border-color);
-    padding-bottom: 3rem;
-    border-top-left-radius: 1rem;
-    border-top-right-radius: 1rem;
-    flex-grow: 1;
+#confirm-transaction #account-details #close-details::after {
+    content: ' ';
```

It does not need to contain a space to become an element.
```suggestion
    content: '';
```